kontrakter/da/varetekt/1.0/KjennelseVaretekt.schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "da/varetekt/1.0/KjennelseVaretekt.schema.json",
+  "$id": "https://politiet.no/da/varetekt/1.0/KjennelseVaretekt.schema.json",
```

Comment:
Jeg ble litt usikker her... I og med at det er domstolen som "eier" denne delen av kontrakten er det kanskje riktigere at domstolen står i urle'en?? Tror jeg vil foreslå _https://domstol.no/da/varetekt/1.0/KjennelseVaretekt.schema.json_
Noen tanker/innsigelser mot det?

Reply:
Enig. Jeg endrer for Rune og merger inn 😄  :+1: